main.js

```diff
@@ -0,0 +1,305 @@
+var addWeather = function (data) {
+  weatherToday = [];
+  var conditionLink = data.weather[0].icon;
+  //checks the icon to set corresponding background color
+  setBackgroundColor(conditionLink);
+  var roundedTemp = Math.round(data.main.temp) + "\xB0";
+  var conditionLinkURL = 'http://openweathermap.org/img/wn/' + conditionLink + '@2x.png';
+  var newCity = {
+    currentCity: data.name || null,
+    currentTemp: roundedTemp || null,
+    currentCondition: data.weather[0].main || null,
+    imageURL: conditionLinkURL || null
+  }
+  weatherToday.push(newCity);
+  renderPage(); 
+};
+//puts 5 day forecast data into an obj, push to array
+var addForecast = function (data) {
+  //take api data, split up into dif arrays for each day of week
+  forecastDataToArrays(data);
+  getDayOfWeek();
+  getEachDayTemp();
+  getEachDayCondition();
+  getEachDayConditionIcon();
+  forecast = [];
+  var newForecast = {
+    firstDayTemp: forecastDataComputed[0][0] || null,
+    firstDay: daysOfWeek[theDay] || null,
+    firstDayCondition: forecastDataComputed[0][1] || null,
+    firstDayimageURL: 'http://openweathermap.org/img/wn/' + forecastDataComputed[0][2] + '@2x.png' || null,
+    secondDayTemp: forecastDataComputed[1][0] || null,
+    secondDay: daysOfWeek[theDay + 1] || null,
+    secondDayCondition: forecastDataComputed[1][1] || null,
+    secondDayimageURL: 'http://openweathermap.org/img/wn/' + forecastDataComputed[1][2] + '@2x.png' || null,
+    thirdDayTemp: forecastDataComputed[2][0] || null,
+    thirdDay: daysOfWeek[theDay + 2] || null,
+    thirdDayCondition: forecastDataComputed[2][1] || null,
+    thirdDayimageURL: 'http://openweathermap.org/img/wn/' + forecastDataComputed[2][2] + '@2x.png' || null,
+    fourthDayTemp: forecastDataComputed[3][0] || null,
+    fourthDay: daysOfWeek[theDay + 3] || null,
+    fourthDayCondition: forecastDataComputed[3][1] || null,
+    fourthDayimageURL: 'http://openweathermap.org/img/wn/' + forecastDataComputed[3][2] + '@2x.png' || null,
+    fifthDayTemp: forecastDataComputed[4][0] || null,
+    fifthDay: daysOfWeek[theDay + 4] || null,
+    fifthDayCondition: forecastDataComputed[4][1] || null,
+    fifthDayimageURL: 'http://openweathermap.org/img/wn/' + forecastDataComputed[4][2] + '@2x.png' || null
+  }
+  forecast.push(newForecast);
+  renderPageForecast();
+};
+//renders current weather onto page
+var renderPage = function () {
+  $('.weather').empty();
+  var source = $('#weather-template').html();
+  var template = Handlebars.compile(source);
+  var newHTML = template(weatherToday[0]);
+ $('.weather').append(newHTML);
+};
+//renders forecast onto page
+var renderPageForecast = function () {
+  $('.weather-two').empty();
+  var source = $('#forecast-template').html();
+  var template = Handlebars.compile(source);
+  var newHTML = template(forecast[0]);
+ $('.weather-two').append(newHTML);
+};
+//takes forecast data and splits into array for each day of week
+var forecastDataToArrays = function (data) {
```

Comment:
A couple of thoughts:

1. This probably isn't the most ideal thing in the world (to hardcode every 8 numbers in a bunch of for loops), but it's an okay first pass as it works.
2. Ideally, this function would return a new data-structure instead of mutating one that exists somewhere else (`daysOfWeekData`). It's just safer that way, cleaner and not so hard to track down all the data that's being mutated and where.